cmd/plugins/memory-qos/main.go

```diff
@@ -0,0 +1,304 @@
+// Copyright 2023 Inter Corporation. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"gopkg.in/yaml.v2"
```

Comment:
nitpick: I habitually use `sigs.k8s.io/yaml` instead of this (or any other) one. Here is the [short rationale](https://pkg.go.dev/sigs.k8s.io/yaml#section-readme) and a [much longer one](http://web.archive.org/web/20190603050330/http://ghodss.com/2014/the-right-way-to-handle-yaml-in-golang/) why this is a good idea.

Reply:
Fixed.